clients/java/src/main/proto/raccoon.proto

```diff
@@ -0,0 +1,157 @@
+syntax = "proto3";
+
+package odpf.raccoon.v1beta1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/odpf/proton/raccoon/v1;raccoonv1";
+option java_multiple_files = true;
+option java_outer_classname = "EventProto";
+option java_package = "io.odpf.proton.raccoon";
+
+service EventService {
```

Comment:
Could we avoid hardcoding this proto here and instead have a pipeline that publishes java generated artifacts as jars from the proton repo?  we could then have this as dependency here. I believe we just need classes for Event/EventWrappers ?

Reply:
Okay, we can add the setup in the proton to generate jar (maven).
I will remove this after the proton repo changes

Reply:
Created the PR for generating the proton repo jar: https://github.com/odpf/proton/pull/220

Reply:
@chakravarthyvp Can you please review the PR except the generated proto files, 
Those I will move once, I import the proto jar. So the PR will be open still then.

Meantime, I will be starting on the proto jar publish.